asio/src/tests/unit/CMakeLists.txt

```diff
@@ -0,0 +1,160 @@
+set(target_prefix asio_unit_)
+
+add_library(asio::standalone ALIAS standalone)
```

Comment:
Why is this aliased here? This should be in the library's project CML, not deeply nested in an incomplete test CML.